core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java

```diff
@@ -121,7 +121,7 @@ public Collection<MetricInfo> getCustomMetrics(GlobalMetricsConfiguration config
          attributes.add(MetricUtils.<RpcManagerImpl>createFunctionTimer("CrossSiteReplicationTimes",
                "Cross Site Replication Times",
                (rpcManager, timerTracker) -> rpcManager.xSiteMetricsCollector.registerTimer(timerTracker),
-               TimeUnit.NANOSECONDS, null));
+               TimeUnit.MICROSECONDS, null));
```

Comment:
Does this also change the output unit in the endpoint?

Reply:
I'm tempted to remove the parameter from the method and store everything in nanoseconds. WDYT?

Reply:
+1. My IDE shows every place is using nanoseconds already, and looking at the diff, it simplifies a lot of places.